uint8 running_phase
uint8[] running_mode
uint8[] state_limit
int64[] state_pulse
float64[] reference
float64[] current
